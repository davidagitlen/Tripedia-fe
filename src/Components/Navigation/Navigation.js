import React from 'react';
import './Navigation.scss';

const Navigation = () => {
  return(
    <p>Navigation</p>
  )
}

export default Navigation;