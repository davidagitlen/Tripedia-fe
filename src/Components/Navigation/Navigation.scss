.navigation__contianer{
  background: whitesmoke;
  height: 20vh;
  display: flex;
  align-items: center;
  justify-content: space-between;
  padding-left: 125px;
  padding-right: 50px;
}
.logo-title__nav {
  display: flex;
  align-items: center;
  justify-content: space-between;
  width: 43%;
}
.tripedia-logo__nav {
  height: 19vh;
  width: 19vh;
  border-radius: 50%;
}
.tripedia-text__nav {
  font-size: 80px;
}