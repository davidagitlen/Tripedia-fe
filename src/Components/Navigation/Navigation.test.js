import React from 'react';
import { shallow } from 'enzyme';
import Navigation from './Navigation';

describe('Navigation', () => {
 
  it("should match the wrapper with data passed in", () => {
    const wrapper = shallow(<Navigation />);
    expect(wrapper).toMatchSnapshot();
  });
});