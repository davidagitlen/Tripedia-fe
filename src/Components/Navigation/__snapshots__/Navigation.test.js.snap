// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation should match the wrapper with data passed in 1`] = `
<nav
  className="navigation__contianer"
>
  <div
    className="logo-title__nav"
  >
    <img
      alt="Tripedia Logo"
      className="tripedia-logo__nav"
      src="Logo1.png"
    />
    <h1
      className="tripedia-text__nav"
    >
      TRIPEDIA
    </h1>
  </div>
  <NavLink
    to="/"
  >
    <p>
      LogOut
    </p>
  </NavLink>
</nav>
`;
